readme.md

```diff
@@ -76,6 +58,8 @@ In the previous commands you created a file called _settings.py_ at the root of
 the project.
 Tune your settings there.
 
+In _install.py_ and _start.py_, `DB_PORT` is an environment variable where your Mongo instance is running. The default for a native install (27017) is preserved. _Lazy installation_ runs on 27018 by default.
```

Comment:
> In _install.py_ and _start.py_, `DB_PORT` is an environment variable where your Mongo instance is running

`DB_PORT` is actually available for all, not limited to install and start scripts. I guess you can therefore get rid of "In _install.py_ and _start.py_, ".

Reply:
Done !